sklearn/manifold/tests/test_mds.py

```diff
@@ -55,10 +55,103 @@ def test_smacof_error():
         mds.smacof(sim, init=Z, n_init=1)
 
 
+def test_svd():
```

Comment:
I think we can remove this test in favor of `test_MDS_svd`, which is largely the same. This one is more of a unit test (which is good) but let's avoid redundancy

Reply:
Done